### ISSUE TITLE
InfiniteScroll does not work in FireFox 

### ISSUE THREAD

**Issue by RobertWeaver** [driftyco/ionic#9596]
**Ionic version:**  (check one with "x")
[ ] **1.x**
[X] **2.x**

**I'm submitting a ...**  (check one with "x")
[X] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or http://ionicworldwide.herokuapp.com/

**Current behavior:**

InfiniteScroll does not work in FireFox using latest nightly.

**Expected behavior:**

InfiniteScroll should work the same as in Chrome on FireFox

**Steps to reproduce:**

http://plnkr.co/edit/9Py3p75apBwG3ECTzbTI?p=preview

**Other information:**

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

```
Ionic Framework Version: 2.0.0-rc.3-201612100458
Ionic CLI Version: 2.1.17
Ionic App Lib Version: 2.1.7
Ionic App Scripts Version: 0.0.45
ios-deploy version: Not installed
ios-sim version: Not installed
OS: Windows 10
Node Version: v6.9.1
Xcode version: Not installed
```

**Comment by Daskus1** [comment]
This is still not working. I am trying to build a PWA with Ionic and want to support all major browsers (one of them clearly is Firefox). 
InfiniteScroll does not work on both mobile and desktop Firefox.
Here my ionic info:
```
Cordova CLI: 6.4.0
Ionic Framework Version: 2.0.0
Ionic CLI Version: 2.2.1
Ionic App Lib Version: 2.2.0
Ionic App Scripts Version: 1.0.0
ios-deploy version: Not installed
ios-sim version: Not installed
OS: Windows 10
Node Version: v4.6.0
Xcode version: Not installed
```

**Comment by godzsa** [comment]
It is not working in Firefox in the API documentation either: http://ionicframework.com/docs/v2/api/components/infinite-scroll/InfiniteScroll/

**Comment by RobertWeaver** [comment]
There is now a pull request to possibly fix this issue. Hopefully soon! :tada:

https://github.com/driftyco/ionic/pull/10752

**Comment by lukasrein97** [comment]
Does it work on Chrome or any Browser else?

**Comment by manucorporat** [comment]
Fixed in 2.3
